driver/src/main/java/com/code_intelligence/jazzer/autofuzz/Meta.java

```diff
@@ -445,27 +454,31 @@ static Object consume(FuzzedDataProvider data, Type genericType, AutofuzzCodegen
       if (visitor != null) {
         throw new AutofuzzError("codegen has not been implemented for Method.class");
       }
-      return data.pickValue(sortExecutables(YourAverageJavaClass.class.getMethods()));
+      return data.pickValue(getAccessibleMethods(YourAverageJavaClass.class));
     } else if (type == Constructor.class) {
       if (visitor != null) {
         throw new AutofuzzError("codegen has not been implemented for Constructor.class");
       }
-      return data.pickValue(sortExecutables(YourAverageJavaClass.class.getConstructors()));
+      return data.pickValue(getAccessibleMethods(YourAverageJavaClass.class));
     } else if (type.isInterface() || Modifier.isAbstract(type.getModifiers())) {
       List<Class<?>> implementingClasses = implementingClassesCache.get(type);
       if (implementingClasses == null) {
-        ClassGraph classGraph =
-            new ClassGraph().enableClassInfo().enableInterClassDependencies().rejectPackages(
-                "jaz.*");
+        ClassGraph classGraph = new ClassGraph()
```

Comment:
That's true. Due to the cache, we will probably not scan that often, but it would still be useful to only scan once. I don't know whether there is a good way to wrap `ScanResult` in something like a `SoftReference` though - I will need to read up on this first.